imports/plugins/included/connectors-shopify/server/methods/api/products.js

```diff
@@ -0,0 +1,46 @@
+import Shopify from "shopify-api-node";
+
+import { Meteor } from "meteor/meteor";
+import { check, Match } from "meteor/check";
+
+import { Reaction, Logger } from "/server/api";
+
+import { getApiInfo } from "./api";
+import { connectorsRoles } from "../../lib/roles";
+
+/**
+ * @file Shopify connector wrapper for api calls for products
+ *       wraps shopify product api calls in reaction methods
+ * @module connectors/shopify/api/products
+ */
+
+export const methods = {
+  /**
+   * Gets a count of the products from Shopify with the API credentials setup for your store.
+   *
+   * @async
+   * @method connectors/shopify/api/products/count
+   * @param {object} options An object of options for the shopify API call. Available options here: https://help.shopify.com/api/reference/product#count
+   * @returns {number} Number of products
+   */
+  async "connectors/shopify/api/products/count"(options) {
+    check(options, Match.Maybe(Object));
+
+    if (!Reaction.hasPermission(connectorsRoles)) {
+      throw new Meteor.Error(403, "Access Denied");
```

Comment:
Is this the correct param format for `Meteor.Error`?

Reply:
Good catch.
I think this is wrong by our style guide now as we don't throw codes anymore and it should be
```
throw new Meteor.Error("access-denied", "Access denied");
```